mephisto/abstractions/providers/prolific/prolific_datastore_tables.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+CREATE_STUDIES_TABLE = """
+    CREATE TABLE IF NOT EXISTS studies (
+        prolific_study_id TEXT PRIMARY KEY UNIQUE,
+        unit_id TEXT,
+        prolific_submission_id TEXT,
+        link TEXT,
+        assignment_time_in_seconds INTEGER NOT NULL,
+        creation_date DATETIME DEFAULT CURRENT_TIMESTAMP
+    );
+"""
+
+CREATE_RUN_MAP_TABLE = """
+    CREATE TABLE IF NOT EXISTS run_mappings (
+        prolific_study_id TEXT,
+        run_id TEXT
+    );
+"""
+
+CREATE_REQUESTERS_TABLE = """
+    CREATE TABLE IF NOT EXISTS requesters (
+        requester_id TEXT PRIMARY KEY UNIQUE,
+        is_registered BOOLEAN
+    );
+"""
+
+CREATE_UNITS_TABLE = """
+    CREATE TABLE IF NOT EXISTS units (
+        unit_id TEXT PRIMARY KEY UNIQUE,
+        is_expired BOOLEAN
+    );
+"""
```

Comment:
These values are stored in the main database, and can otherwise be queried from Prolific or local state, so no need to record them here too